<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MopViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/moody.png</normaloff>:/icons/resources/moody.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="glContainer"/>
    </item>
    <item>
     <widget class="QGroupBox" name="AxisControlsBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Axis</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="AxisSliderX">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="AxisSliderY">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="AxisSliderZ">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_Mop_File">
   <property name="text">
    <string>&amp;Open Mop File</string>
   </property>
  </action>
  <action name="action_CLose_Mop_FIle">
   <property name="text">
    <string>&amp;Close Mop FIle</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Show_Controls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Controls</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
